internal/oem/oem.go

```diff
@@ -61,6 +62,21 @@ func (c Config) NewFetcherFunc() providers.FuncNewFetcher {
 	}
 }
 
+// Status takes b as whether ignition passes (true) or fails
+// msg is the error or success message
+func (c Config) Status(b bool, msg string) providers.FuncStatusFetcher {
```

Comment:
there shouldn't be packet specific stuff in `oem/oem`. Instead the packet version should be registered below.